3rdparty/python/requirements.txt

```diff
@@ -39,6 +39,9 @@ typing-extensions==4.0.1
 mypy-typing-asserts==0.1.1
 
 
+# This dependency is only for debugging Pants itself, and should never be imported
```

Comment:
Maybe worth a quick `ValidateDependenciesRequest` implementation in `pants-plugins/internal_plugins` to enforce this? https://github.com/pantsbuild/pants/issues/13393#issuecomment-1160348190

Reply:
Given that, I'm fine with validating both of these the same way once they're both plugins.